### ISSUE TITLE
RT shadows artifacts

### ISSUE THREAD

**Issue by ershovdz** [turanszkij/WickedEngine#578]
Hello, 
Is it correct behavior of the shadows (flickering)?
Link to [Youtube](https://www.youtube.com/watch?v=X8Wzc-gZuTY)

**Comment by turanszkij** [comment]
No, it is not correct. I knew about this issue, but don't know yet why it happens.

**Comment by turanszkij** [comment]
It's been fixed.
